packages/core/feature-flags/README.mdx

```diff
@@ -0,0 +1,158 @@
+
+1. **Single evaluation**: performs the evaluation once, and doesn't react to updates. These APIs are synchronous in the
+browser, and asynchronous in the server.
+2. **Observed evaluation**: observes the flag for any changes so that the code can adapt. These APIs return an RxJS observable.
+
+Also, the APIs are typed, so you need to use the appropriate API depending on the `variationType` you defined your flag:
+
+|  Type   | Single evaluation                                       | Observed evaluation                                      |
+|:-------:|:--------------------------------------------------------|:---------------------------------------------------------|
+| Boolean | `core.featureFlags.getBooleanValue(flagName, fallback)` | `core.featureFlags.getBooleanValue$(flagName, fallback)` |
+| String  | `core.featureFlags.getStringValue(flagName, fallback)`  | `core.featureFlags.getStringValue$(flagName, fallback)`  |
+| Number  | `core.featureFlags.getNumberValue(flagName, fallback)`  | `core.featureFlags.getNumberValue$(flagName, fallback)`  |
+
+### Request handler context
+
+Additionally, to make things easier in our HTTP handlers, the _Single evaluation_ APIs are available as part of the core
+context provided to the handlers:
+
+```typescript
+async (context, request, response) => {
+  const { featureFlags } = await context.core;
+  return response.ok({
+    body: {
+      number: await featureFlags.getNumberValue('example-number', 1),
+    },
+  });
+}
+```
+
+## Extending the evaluation context
+
+The <DocLink id="kibCloudExperimentsPlugin" section="evaluation-context" text="current evaluation context"/> should have
+enough information to declare the segmentation rules for your feature flags. However, if your use case requires additional
+context, feel free to call the API `core.featureFlags.setContext()` from your plugin.
+
+At the moment, we use 2 levels of context: `kibana` and `organization` that we can use for segmentation purposes at
+different levels. By default, the API appends the context to the `kibana` scope. If you need to extend the `organization`
+scope, make sure to add `kind: 'organization'` to the object provided to the `setContext` API.
+
+## Config overrides
+
+To help with testing, and to provide an escape hatch in cases where the flag evaluation is not behaving as intended,
+the Feature Flags Service provides a way to force the values of a feature flag without attempting to resolve it via the
+provider. In the `kibana.yml`, the following config sets the overrides:
+
+```yaml
+feature_flags.overrides:
+  my-feature-flag: 'my-forced-value'
+```
+
+> [!WARNING]  
+> There is no validation regarding the variations nor the type of the flags. Use these overrides with caution.
+
+### Dynamic config
+
+When running in our test environments, the overrides can be updated without restarting Kibana via the HTTP `PUT /internal/core/_settings`:
```

Comment:
Just for illustration purposes. I don't think that we can use that API in the Console because it doesn't allow the version to be specified via query param (it doesn't have `enableQueryVersion: true`)

Although maybe we should allow that? 🤔
Nah... it's for testing purposes only... if anyone wants use it outside of automated tests, they can always use `curl`